diff --git a/prebuild/apk/Android.mk b/prebuild/apk/Android.mk
index f43f79b..af25226 100755
--- a/prebuild/apk/Android.mk
+++ b/prebuild/apk/Android.mk
@@ -42,14 +42,14 @@ include $(BUILD_PREBUILT)
 
 ####################################
 
-include $(CLEAR_VARS)
-LOCAL_MODULE := AWUpdate
-LOCAL_MODULE_TAGS := optional
-LOCAL_CERTIFICATE := PRESIGNED
-LOCAL_MODULE_CLASS := APPS
-LOCAL_MODULE_SUFFIX := $(COMMON_ANDROID_PACKAGE_SUFFIX)
-LOCAL_SRC_FILES := $(LOCAL_MODULE).apk
-include $(BUILD_PREBUILT)
+#include $(CLEAR_VARS)
+#LOCAL_MODULE := AWUpdate
+#LOCAL_MODULE_TAGS := optional
+#LOCAL_CERTIFICATE := PRESIGNED
+#LOCAL_MODULE_CLASS := APPS
+#LOCAL_MODULE_SUFFIX := $(COMMON_ANDROID_PACKAGE_SUFFIX)
+#LOCAL_SRC_FILES := $(LOCAL_MODULE).apk
+#include $(BUILD_PREBUILT)
 
 ####################################
 
